jerry-debugger/src/lib/cdt-proxy-http.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ChromeDevToolsProxyServer, JERRY_DEBUGGER_VERSION, DEVTOOLS_PROTOCOL_VERSION } from './cdt-proxy';
+import * as http from 'http';
+import * as path from 'path';
+
+export function onHttpRequest(this: ChromeDevToolsProxyServer,
+                              request: http.IncomingMessage, response: http.ServerResponse) {
+  let url = request.url || '';
```

Comment:
I had some additional comments on this function. Perhaps you missed them because of Github's "outdated" comment folding?

https://github.com/martijnthe/jerryscript/pull/1/commits/d7d4b6c8d60eb43383cd179a626dfbfb2b1d130f#r168715705

Reply:
Yeah, I did miss some apparently. I'll go back through them now...